introduction_to_amazon_algorithms/jumpstart_text_embedding/Amazon_JumpStart_Text_Embedding.ipynb

```diff
@@ -0,0 +1,496 @@
+{
```

Comment:
<p>If you want to keep this image, please include it as a standalone JPG/PNG that is displayed in the notebook, rather than embedded as a cell output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>